.ci.yaml

```diff
@@ -3653,6 +3653,16 @@ targets:
         ["devicelab", "ios", "mac"]
       task_name: flutter_gallery_ios__start_up
 
+  - name: Mac_ios flutter_gallery_ios__start_up_xcode_debug
```

Comment:
Do we want these tests in the release branch?

Reply:
As discussed offline, these tests are in bringup and therefore should be okay